migrations/versions/60a681c463a2_create_donors_override.py

```diff
@@ -0,0 +1,33 @@
+"""create_donors_override
+
+Revision ID: 60a681c463a2
+Revises: 971ddb205e0c
+Create Date: 2021-05-25 12:50:58.779359
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "60a681c463a2"
+down_revision = "971ddb205e0c"
```

Comment:
```suggestion
down_revision = "467ee396a68e"
```
Tohle PR mergneme hned jako další, takže slibuji, že se tohle mění naposledy.